docs(ske-frontend): added documentation for SKE frontend plugin (#579)
syntasso/kratix-docs: docs/ske/10-integrations/10-backstage/03-installation/02-plugins.mdx

);
```

The matcher selects Promise and Resource Components using either the `kratix.io/type` annotation or `spec.type`. For example, a Resource can use `kratix.io/type: resource` or
`spec.type: kratix-resource`.

:::warning

Add `kratix.io/type: promise` to every Promise Component. Although the route
matcher recognises `spec.type: kratix-promise`, the page also needs the
annotation when it loads Promise data. Without it, the Promise Overview does
not load. It shows a missing-annotation message for Resource fields instead of
the Promise details.

:::

### Plugin components

#### Promise Overview

`KratixPromiseEntityPage` renders a dedicated Software Catalog page for
Promise entities:

- Name and API details for the service that users can create
- Promise description and current status
- Create button linking to the Software Template for new service instances
- Dependency graph showing related instances

#### Resource Overview

`KratixResourceEntityPage` renders a dedicated Software Catalog page for
Resource entities:

> - Instance name, namespace, type, and the Promise from which it was created

Reviewer: Will keep this as Resource or change it to Resource Entity.

Author: ```suggestion
- Resource name, namespace, kind, and parent Promise
```

- Optional Promise summary (markdown-rendered)
- Current instance status

> - Dependency graph showing which Promise the instance came from

Author: ```suggestion
- Current resource status
- Dependency graph showing the relationship to the parent Promise
```

#### Resource Manage

> The **Manage** tab on `KratixResourceEntityPage` provides controls for an
> existing instance:

Author: ```suggestion
The **Manage** tab on `KratixResourceEntityPage` provides Resource lifecycle actions:
```

> - Pre-filled update form with the instance's current settings (`spec`)

Reviewer: think we should keep this as Resource's

Author: ```suggestion
- Pre-filled update form with the Resource's current spec
```
